### PR TITLE
cleanup: fix all improper uses of MxResult

### PR DESCRIPTION
**Pull request by Ramen2X**
This PR fixes all uses of MxResult to use its' defined names rather than hardcoded values.

### CODE REVIEW

**Comment by Ramen2X** [comment]
Also noticed `MXIOINFO::Seek()` was returning an MxLong despite sticking to the MxResult standard, so I've corrected this too. Match of the function remains the same.

File: LEGO1/mxioinfo.cpp [hunk 1]
```diff
@@ -124,9 +124,9 @@ MxLong MXIOINFO::Read(void *p_buf, MxLong p_len)
 }
 
 // OFFSET: LEGO1 0x100cca00
-MxLong MXIOINFO::Seek(MxLong p_offset, int p_origin)
+MxResult MXIOINFO::Seek(MxLong p_offset, int p_origin)
```
Comment (foxtacles) [inline LEGO1/mxioinfo.cpp:127]: This should remain `MxLong`. If you look at the function you will see it assigns various values to `result` which don't necessarily have to be `SUCCESS` or `FAILURE`, so using `MxResult` as the return type would be misleading

Comment (Ramen2X) [reply LEGO1/mxioinfo.cpp:127]: Sorry! I didn't notice this. I think I was tripped up by the `result` name.

Comment (foxtacles) [reply LEGO1/mxioinfo.cpp:127]: I'd just leave it `-1` and only use the constants in the context of the `MxResult` type (think C++11 `enum class` behavior)

**Review by foxtacles — APPROVED** [review]

**Pull request merged**